mllib/src/main/scala/org/apache/spark/mllib/regression/StreamingLinearAlgorithm.scala

```diff
@@ -130,7 +147,10 @@ abstract class StreamingLinearAlgorithm[
   }
 
 
-  /** Java-friendly version of `predictOnValues`. */
+  /** Java-friendly version of `predictOnValues`.
```

Comment:
This is mis-formatted; the line with the `/**` shouldn't have the method description on it (see the other multi-line Scaladoc strings in this file for examples of the proper style).